src/main/java/com/oltpbenchmark/catalog/Table.java

```diff
@@ -91,7 +91,7 @@ public void addIndex(Index index) {
 
     public Index getIndex(String indexName) {
         for (Index catalog_idx : this.indexes) {
-            if (catalog_idx.getName().equalsIgnoreCase(indexName)) {
+            if (indexName.equalsIgnoreCase(catalog_idx.getName())) {
```

Comment:
Why does this matter?

Reply:
When index type is `tableIndexStatistic`, index name is null, and we end up hitting a NPE.

I just worked around the issue at the time without exploring it much and I didn't even realize that this change was part of the PR :) I was examining the [DatabaseMetaData interface](https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getIndexInfo(java.lang.String,%20java.lang.String,%20java.lang.String,%20boolean,%20boolean)) definition and it seems this is the expected behavior by all implementers:
```
INDEX_NAME String => index name; null when TYPE is tableIndexStatistic
```
I think the proper solution should be ignoring the entry in the loop if type is `tableIndexStatistic`. I'll change the code. Is it fine to do it as part of this PR or you prefer I open a new one?

Reply:
You can do it in this one.